core/src/main/scala/cats/SemigroupK.scala

```diff
@@ -67,4 +67,18 @@ import simulacrum.typeclass
     new ComposedSemigroupK[F, G] {
       val F = self
     }
+
+  /**
+   * Combines `F[A]` and `F[B]` into a `F[Either[A,B]]]`.
+   *
+   * Example:
+   * {{{
+   * scala> import cats.SemigroupK
+   * scala> import cats.data.NonEmptyList
+   * scala> SemigroupK[NonEmptyList].sum(NonEmptyList.one(1), NonEmptyList.one(2))
```

Comment:
This is nitpicky, but what do you think about using different types for this example? Otherwise looks good to me.

Reply:
I was hesitant about using NonEmptyList here, but I wasn't sure about the alternatives either, so I stayed with NEL.

do you perhaps have any suggestions which type could be used here instead? :)

Reply:
@pk044 Oh, sorry, that wasn't very clear. I think `NonEmptyList` is a good choice, just that using different values for `A` and `B` might make it a little clearer—e.g. something like:
```scala
SemigroupK[NonEmptyList].sum(NonEmptyList.one("abc"), NonEmptyList.one(2))
```

Reply:
ok, i changed the value :+1: